src/ethereum/types/index.ts

```diff
@@ -20,3 +20,5 @@ export type Transaction = {
 };
 
 export type ValidatorStatus = 'unknown' | 'pending' | 'deposited';
+
+export type ValidatorCode = '0' | '1' | string;
```

Comment:
we need strict type here. 

@Messer4 Can we assume that? 
``enum ValidatorStatus {
  Unknown = 0, 
  Pending = 1, 
  Deposited = 2 // Assuming '2' represents deposited status
}``